### PR TITLE
Install jest enzyme

### PR DESCRIPTION
**Pull request by abechoe**
This PR installs and configures jest and enzyme, for testing the application.

### CODE REVIEW

File: src/setupTests.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import { configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+configure({adapter: new Adapter() });
```
Comment (yirmiyahu) [inline src/setupTests.js:4]: nitpick `({ adapter`

Comment (yirmiyahu) [reply src/setupTests.js:4]: Should there be a spec or test dir?

File: package.json [hunk 2]
```diff
@@ -4,11 +4,17 @@
...
   "license": "MIT",
   "devDependencies": {
-    "@babel/core": "^7.0.0-beta.56",
-    "@babel/preset-env": "^7.0.0-beta.56",
-    "@babel/preset-react": "^7.0.0-beta.56",
-    "babel-loader": "^8.0.0-beta",
+    "@babel/core": "^7.0.0",
+    "@babel/preset-env": "^7.0.0",
+    "@babel/preset-react": "^7.0.0",
+    "babel-core": "^7.0.0-0",
```
Comment (yirmiyahu) [inline package.json:10]: Were the babel packages upgraded by dependency of jest/enzyme?

File: src/setupTests.js (hunk 1 above)
Comment (abechoe) [reply src/setupTests.js:4]: Yes, the default test dir is `src/__tests__`. I created the directory locally, but it's empty atm, so it wasn't picked up by git.

File: package.json (hunk 2 above)
Comment (abechoe) [reply package.json:10]: No, I manually updated the package versions for babel. When I tried installing `babel-jest` with the versions of the babel packages that we had before, it gave a warning about an unmet peer dependency for babel. I was following this step: https://jestjs.io/docs/en/getting-started#using-babel

**Review by yirmiyahu — APPROVED** [review]

**Pull request merged**